lib/tutorials/en_US/server-methods.md

```diff
@@ -1,21 +1,40 @@
-## Server methods
+# Server Methods
 
 _This tutorial is compatible with hapi v17_
 
-Server methods are a useful way of sharing functions by attaching them to your server object rather than requiring a common module everywhere it is needed. To register a server method, you call [`server.method()`](https://hapijs.com/api#server.method()). There are two different ways to call this function. You can call it with the signature `server.method(name, method, [options])`, for example:
+- [Overview](#overview)
+- [server.method()](#server.method)
+    - [Name](#name)
+    - [Method](#method)
+- [Options](#options)
+    - [Cache](#cache)
+    - [Generate a Custom Key](#key)
+    - [Bind](#key)
+- [server.methods()](#server.methods)
 
-```javascript
+
+
+## <a name="overview" /> Overview
+
+Server methods are a useful way of sharing functions by attaching them to your server object rather than requiring a common module everywhere it is needed. To register a server method, you call [`server.method()`](https://hapijs.com/api#server.method()). There are two different ways to call this function. You can call it with the signature `server.method(name, method, [options])`, or you can call it with the signature `server.method(method)`, where `method` is an object with `name`, `method`, and `options` parameters (note that you may also pass an array of these objects).
```

Comment:
It might be nice to make mention of caching in the overview.

Reply:
Will do, on it